tests/test_diffraction_objects.py

```diff
@@ -468,10 +468,61 @@ def test_dump(tmp_path, mocker):
 
 
 @pytest.mark.parametrize(
-    "do_init_args, expected_do_dict, divide_by_zero_warning_expected",
+    "do_init_args, expected_do_dict, divide_by_zero_warning_expected, wavelength_warning_expected",
     [
         # Test __dict__ of DiffractionObject instance initialized with valid arguments
-        (  # C1: Minimum arguments provided for init, expect all attributes set without None
+        (  # C1: Instantiate DO with empty arrays, expect it to be a valid DO, but with everything empty
```

Comment:
basically added the two C1 C2 back which  you've provided here https://github.com/diffpy/diffpy.utils/issues/241